gnssanalysis/filenames.py

```diff
@@ -306,9 +306,12 @@ def nominal_span_string(span_seconds: float) -> str:
             unit = "W"
             span_unit_counts = int(span_seconds // gn_const.SEC_IN_WEEK)
     elif span_seconds >= gn_const.SEC_IN_WEEK:
-        if (span_seconds % gn_const.SEC_IN_WEEK) < gn_const.SEC_IN_DAY:
+        num_weeks = int(span_seconds // gn_const.SEC_IN_WEEK)
```

Comment:
This is a good idea, makes things a lot clearer for the `if` statement that comes next